Sources/TuistSupport/Logging/Logger.swift

```diff
@@ -53,6 +57,9 @@ public enum LogOutput {
             handler = StandardLogHandler.self
         case .json:
             handler = JSONLogHandler.self
+        case .quiet:
+            LoggingSystem.bootstrap(quietLogHandler(label:))
+            return
```

Comment:
```suggestion
            handler = StandardLogHandler(label: label, logLevel: .warning)
```

Shouldn't this work as well?

In the initial issue, I was proposing to only include `error` logs – but I'm not against surfacing warnings as well. Worth asking for some extra opinions on this or find examples of how `--quiet` works in other tools.

Reply:
I can confirm that `swift` (SPM) surfaces the warnings: 
<img width="1264" alt="Screenshot 2024-11-08 at 11 10 36" src="https://github.com/user-attachments/assets/40dd1cd9-083f-4e7e-b78c-74b8cf20bd9f">

Reply:
Should we also surface the `.notice` and `.success` level? That's what they seem to be doing – I assume `Build complete!` is not a warning.

Let's say for the following `tuist generate` example output:
```
Loading and constructing the graph (Notice level)
It might take a while if the cache is empty (Info level)
Fetching remote binaries. Hold tight... (Info level)
Finished fetching remote binaries (Info level)
Using cache binaries for the following targets: CNIOLinux (Info level)
Generating workspace Tuist.xcworkspace (Notice level)
Generating project TuistApp (Info level)
Generating project Tuist (Info level)
Project generated. (Success level)
Total time taken: 6.503s (Info level)

The following warnings need attention: (Warning level)
 · We detected outdated dependencies. Please run "tuist install" to update them. (Warning level)
```

I'd expect the following output when run with `tuist generate --quiet`:
```
Loading and constructing the graph (Notice level)
Generating workspace Tuist.xcworkspace (Notice level)
Project generated. (Success level)

The following warnings need attention: (Warning level)
 · We detected outdated dependencies. Please run "tuist install" to update them. (Warning level)
```

How does that sound? @tuist/community-core any opinions on this one?